src/otx/core/data/module.py

```diff
@@ -62,18 +62,15 @@ def __init__(
         auto_num_workers: bool = False,
         device: DeviceType = DeviceType.auto,
         input_size: int | tuple[int, int] | None = None,
+        adaptive_input_size: Literal["auto", "downscale", "none"] = "none",
```

Comment:
Can we consider this?
```suggestion
        adaptive_input_size: Literal["auto", "downscale"] | None = None,
```

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3788/commits/0adb7ea54bb57b761f75b423167b45772d08d416